data/data_download/gadm_3.6/Makefile

```diff
@@ -6,7 +6,7 @@ MarxanUser:=$(shell psql -X -A -t "postgresql://$$API_POSTGRES_USER:$$API_POSTGR
 
 import: data/gadm/gadm36_simp.geojson
 	ogr2ogr -makevalid \
-		-overwrite \
+		-update -append \
```

Comment:
I suppose this is the change that avoids having ogr2ogr change our table structure?

That's good. However, I am also wondering if it may be wise to treat each upstream dataset version as a separate subset, both when we insert it via ogr2ogr *and* when we extract subsets (or dump everything)... I fear that if we don't do this *at some point*, it would be hard to keep things predictable in terms of new dataset releases/dataset updates...

I wonder if we should keep past releases of datasets intact after insertion altogether, so also keeping existing scenarios that use this data linked to whatever version of the dataset they are already using. Or at the very least record in a table a list of source datasets and for each of them the upstream release version/date, when they were inserted, etc. Not something for now and paying attention to not overengineer this, but also to not underengineer this for realistic use cases and to avoid brittleness.